WooCommerce/Classes/Authentication/Jetpack Setup/Native Jetpack Setup/LoginJetpackSetupView.swift

```diff
@@ -0,0 +1,174 @@
+import SwiftUI
+
+/// Hosting controller for `LoginJetpackSetupView`.
+///
+final class LoginJetpackSetupHostingController: UIHostingController<LoginJetpackSetupView> {
+    init(siteURL: String, connectionOnly: Bool) {
+        let viewModel = LoginJetpackSetupViewModel(siteURL: siteURL, connectionOnly: connectionOnly)
+        super.init(rootView: LoginJetpackSetupView(viewModel: viewModel))
+    }
+
+    @available(*, unavailable)
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureNavigationBarAppearance()
+    }
+
+    /// Shows a transparent navigation bar without a bottom border.
+    private func configureNavigationBarAppearance() {
+        let appearance = UINavigationBarAppearance()
+        appearance.configureWithTransparentBackground()
+        appearance.backgroundColor = .systemBackground
+
+        navigationItem.standardAppearance = appearance
+        navigationItem.scrollEdgeAppearance = appearance
+        navigationItem.compactAppearance = appearance
+
+        let title = NSLocalizedString("Cancel", comment: "Button to dismiss the site credential login screen")
+        navigationItem.leftBarButtonItem = UIBarButtonItem(title: title, style: .plain, target: self, action: #selector(dismissView))
+    }
```

Comment:
Nit: Please consider reusing this code by moving this to an extension.

Reply:
Good idea - updated in d5558c5e31d823bc35ea318b011a79f9eac42af2.